ddd-koans/Value Object/Self Validation/task.html

```diff
@@ -0,0 +1,27 @@
+<html>
+<p>
+    <b>TimeSignature</b> is described using two properties, the same you have declared in previous task. <br/>
+    However, values of this properties cannot be random integers. There are <b>domain rules</b>.
+    If you don't remember, here you go:
+    <ul>
+        <li>numerator is a number between 1 and 32,</li>
+        <li>denominator is a number between 1 and 32,</li>
+        <li>denominator is a number of power of two.</li>
```

Comment:
I wound if we would like to introduce VOs like `Denominator` and `Numerator`? It sounds reasonable, however, if the `beat` and `note` concepts were introduced, then the design would be different.